app/src/main/res/values-ta/strings.xml

```diff
@@ -0,0 +1,336 @@
+    <string name="d_checkCredentials_errorMessage_noAccess">பநிஇ க்கு அணுகல் இல்லை</string>
+    <string name="d_checkCredentials_errorMessage_noAccessWithMessage">பநிஇ க்கு அணுகல் இல்லை: %s</string>
+    <string name="d_checkCredentials_errorMessage_unknown">தெரியாத பிழை</string>
+    <string name="d_checkCredentials_errorMessage_unknownWithMessage">தெரியாத பிழை: %s</string>
+    <string name="wrongUsernameOrPassword_errorMessage">உள்நுழைய முடியவில்லை. பயனர்பெயர் அல்லது கடவுச்சொல் தவறாக இருக்கலாம்?</string>
+    <string name="previousUpdateNotFinished">முந்தைய புதுப்பிப்பு இன்னும் முடிக்கப்படவில்லை</string>
+    <string name="feedName_favorites">பிடித்தவை</string>
+    <string name="feedName_archived">காப்பகப்படுத்தப்பட்டது</string>
+    <string name="feedName_unread">படிக்காதது</string>
+    <string name="articleList_favoriteMark">பிடித்த குறி</string>
+    <string name="articleList_ArchivedMark">காப்பகப்படுத்தப்பட்ட குறி</string>
+    <string name="noPreviousArticle">தற்போதைய பட்டியலில் முந்தைய கட்டுரை எதுவும் இல்லை</string>
+    <string name="scrolledOverBottom">உருட்டல் %d அதிக நேரம் (கள்) கட்டுரையைப் படிக்க கீழே குறிக்கவும்</string>
+    <string name="markedAsRead">கட்டுரை வாசிப்பு என குறிக்கப்பட்டுள்ளது</string>
+    <string name="noNextArticle">தற்போதைய பட்டியலில் மேலும் கட்டுரைகள் இல்லை</string>
+    <string name="themeName_light">ஒளி</string>
+    <string name="themeName_light_contrast">ஒளி (உயர் மாறுபாடு)</string>
+    <string name="themeName_eink">மின்-மை காட்சிகளுக்கு</string>
+    <string name="themeName_dark">இருண்ட</string>
+    <string name="themeName_dark_contrast">இருண்ட (உயர் மாறுபாடு)</string>
+    <string name="themeName_solarized">சோலரிச்</string>
+    <string name="menu_readArticle_increaseFontSize">எழுத்துரு அளவை அதிகரிக்கவும்</string>
+    <string name="menu_readArticle_decreaseFontSize">எழுத்துரு அளவைக் குறைக்கவும்</string>
+    <string name="aboutText"><b> ஆண்ட்ராய்டுக்கான வாலபாக் பயன்பாடு </b>\n வாலபாக் ஒரு சுய-ஓச்டபிள் பயன்பாடாகும், இது இனி எந்த உள்ளடக்கத்தையும் தவறவிடக்கூடாது. உங்கள் ஓய்வு நேரத்தில் பிரித்தெடுக்கப்பட்ட உள்ளடக்கத்தை சொடுக்கு செய்து, சேமிக்கவும் படிக்கவும்.</string>
+    <string name="readSpeed">வேகத்தைப் படியுங்கள்</string>
+    <string name="voiceHeight">குரல் சுருதி</string>
+    <string name="readVolume">தொகுதி படிக்கவும்</string>
+    <string name="menuDownloadAsFile">கோப்பாக பதிவிறக்கவும்</string>
+    <string name="dialog_title_downloadFileFormat">கோப்பு வடிவமைப்பைத் தேர்ந்தெடுக்கவும்</string>
+    <string name="downloadAsFilePathStart">கட்டுரை பதிவிறக்குகிறது</string>
+    <string name="downloadAsFileArticleDownloaded">கட்டுரை பதிவிறக்கம் செய்யப்பட்டது</string>
+    <string name="downloadAsFileArticleDownloadedDetail">கட்டுரைகள் பதிவிறக்கம்</string>
+    <string name="downloadAsFileTouchToOpen">திறக்கத் தொடவும்</string>
+    <string name="downloadAsFileProgressDetail">வாலபாக் உங்கள் கட்டுரையை %1 $ s %2 $ s இல் பதிவிறக்குகிறது</string>
```

Comment:
looks like this is the problematic line of the translation

Reply:
it was mentioned in the CI output some lines above compared to my previous quote of the CI output:

`/home/runner/work/android-app/android-app/app/src/main/res/values-ta/strings.xml:115:4: Multiple substitutions specified in non-positional format of string resource string/downloadAsFileProgressDetail. Did you mean to add the formatted="false" attribute?`

Reply:
Ok, this is solved now, but wasn't causing the CI error